lib/internal/worker.js

```diff
@@ -294,10 +311,35 @@ function pipeWithoutWarning(source, dest) {
   dest._maxListeners = destMaxListeners;
 }
 
+const resourceLimitsArray = new Float64Array(kTotalResourceLimitCount);
```

Comment:
Is there a reason not to reuse `resourceLimitsRaw` here?

Although if we expose `makeResourceLimits` from this file and calculate the publicly exposed `resourceLimits` from `lib/worker_threads.js` instead it makes more sense to use two arrays in case someone created a worker internally with a resource limit..somehow.

Reply:
Practically speaking, yes, the reason is that `resourceLimitsRaw` is currently not defined in the main thread. And given that this is a very small typed array, I’m okay with that. If you’re concerned about the extra resource usage, I’d probably prefer changing this to not be a static variable and instead generate one for each call to `parseResourceLimits()`?